src/set.coffee

```diff
@@ -78,7 +78,11 @@ install = (_, jasmine) ->
             cachedId = null
             cachedResult = null
             oncePerSuiteWrapper = ->
-              id = jasmine?.getEnv()?.currentSpec?.id || globalPatches.__autoIncrement++
+              specId = jasmine?.getEnv()?.currentSpec?.id
+              if specId == 0
```

Comment:
oy, good catch. really this bug is due to me using `||`, which does truthy conversion (`0 -> false`), instead of `?`, which does an existential compare (`== null`) that avoids converting.

So a better fix is just: `id = jasmine?.getEnv()?.currentSpec?.id ? globalPatches.__autoIncrement++`